src/Security/Authentication/Handler/FailureHandler.php

```diff
@@ -0,0 +1,63 @@
+<?php
```

Comment:
strict_types=1?

Reply:
Perhaps it's the right moment to do this throughout the entire library

Reply:
Good point, I have it as a mission to write all new code strict. And refactor the old code to the strict standard whenever I touch it. So I will add the declaration to those classes.